lib/surface/components/form/text_input.ex

```diff
@@ -7,15 +7,18 @@ defmodule Surface.Components.Form.TextInput do
   property field, :string, required: true
   property value, :string, default: ""
   property class, :css_class
+  property opts, :keyword, default: []
 
   def render(assigns) do
     ~H"""
     {{
       text_input(
         String.to_atom(@form),
         @field,
-        value: @value,
-        class: @class
+        [
+          value: @value,
+          class: @class,
```

Comment:
I believe `text_input/3` can handle `class` and `value` as `nil`, right? Otherwise, we'll need to conditionally add them to `otps`.

Reply:
Not sure what mean. The first test for text_input creates one where I'm not passing a class or value and it works as expected. I actually just added a commit to remove the default for `value`, exactly so that it can be nil and `text_input` will ignore it. Is that what you meant?

Reply:
Yes! Awesome.